### PR TITLE
📈 2315 Update Program Dashboard Invalid Data display

### PR DESCRIPTION
**Pull request by demariadaniel**
# Description of changes

Donor Summary:
- Update Error Banner Text
- Remove Release Status Error icons
- Remove Summary Error Banner
- Change Invalid Donors label copy
- Add Alerts Column w/ Link to Submitted Data
- Alerts Column Filters: API update https://github.com/icgc-argo/platform-api/pull/612

## Type of Change

- [ ] Bug
- [ ] Dependency updates
- [ ] Feature
- [ ] Refactoring
- [ ] Release candidate
- [x] Styling
- [ ] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [x] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [ ] No back-end changes
- [x] Manually tested changes
- [x] Added copyrights to new files
- [x] Connected ticket to PR

### CODE REVIEW

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx [hunk 1]
```diff
@@ -126,14 +126,11 @@ const DonorSummaryTable = ({
     filterState.find((x) => x.field === field)?.values;
 
   const StatusColumnCell = ({ original }: { original: DonorSummaryRecord }) => {
-    const theme = useTheme();
-    const displayIcon = original.validWithCurrentDictionary ? (
+    const displayIcon = (
```
Comment (samrichca) [inline components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:129]: i would remove the `displayIcon` constant and use the component directly

Comment (demariadaniel) [reply components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx]: Yes, updated, this was leftover from when this component was using a ternary

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx [hunk 2]
```diff
@@ -886,6 +883,56 @@ const DonorSummaryTable = ({
...
+                ?.entity || '';
+
+            const linkUrl = urlJoin(
+              `/submission/program/`,
+              programShortName,
+              `/clinical-data/?donorId=${original.donorId}`,
+              errorTab && `&tab=${errorTab}`,
+            );
+
+            return !original.validWithCurrentDictionary ? (
```
[the remark this answers is not in the document]
Comment (demariadaniel) [reply components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:922]: Sure positive condition first is a good rule of thumb, updated

**Review by samrichca — APPROVED** [review]
looking good!

**Pull request merged**